File: example/typescript/class/super-class.ts
class S {
  protected v
  static f = 0
  get g() {
    return 32
  }
  set s(x) {
    this.v = 2;
  }
  m(a, b) {
    return 91
  }
}

class C extends S {
  // 注释的代码存在编译错误
  
  // v() {
  //   return 1  // 非法
  // }
  // f() {
  //   return 2  // 通过：ts并不会为静态变量默认设置一个getter
  // }
  // g() {
  //   return 100  // 非法
  // }
  // s(y) {
  //   return 200  // 非法
  // }
  // m(a, b, c) {
  //   return 92 // 非法
  // }
}

File: example/dart/class/super-class.dart
class S {
  var v;
  final f = 0;
  get g => 32;
  set s(x) => v =2;
  m(a, b) => 91;
}

class C extends S {
  // 注释的代码存在编译错误
  
  // v() => 1; // 非法：方法v()重写隐含的getter方法v
  // f() => 2; // 非法：方法f()重写隐含的getter方法f
  // g() => 100; // 非法：方法g()重写getter方法g
  // s(y) => 200;  // 非法：方法s()重写setter方法s
  // m(a, b, c) => 92; // 警告：重写方法参数个数不一致
}